src/lib/journey/callbacks/password/base.svelte

```diff
@@ -89,8 +89,10 @@
     slot="input-button"
     type="button"
   >
-    <EyeIcon classes="tw_password-icon dark:tw_password-icon_dark" visible={isVisible}
-      ><T key="showPassword" /></EyeIcon
+    <EyeIcon
+      aria-label="show-password"
```

Comment:
I'm assuming there's a reason you added this `aria-label`, but I would think this would be redundant since we are also passing a `title` attribute to the `EyeIcon`. All icons use the `<slot />` mechanism for this. You can see it here with the `EyeIcon`: https://github.com/cerebrl/forgerock-web-login-framework/blob/main/src/lib/components/icons/eye-icon.svelte#L19.

If the `title` attribute is not working well enough in your testing, then I'd like to change this underlying implementation, rather than sticking an `aria-label` on the element as doing so may end up duplicating what's read to the user.